product_abc_classification_sale_stock/models/abc_classification_profile.py

```diff
@@ -333,9 +334,7 @@ def _log_history(self, sale_stock_data_list):
         table = self.env["abc.sale_stock.level.history"]._table
         columns = sale_stock_data._get_col_names()
         self.env.cr.copy_from(vals, table, columns=columns, sep=";")
-        self.env["abc.classification.product.level"].invalidate_cache(
-            ["sale_stock_level_history_ids"]
-        )
+        self.env.invalidate_all()
```

Comment:
you invalidate too much fields....
```suggestion
        self.env["abc.classification.product.level"].invalidate_model(
             ["sale_stock_level_history_ids"]
        )
        self.env["product.template"].invalidate_model(
             ["abc_classification_product_level_ids"]
        )
        self.env["product.product"].invalidate_model(
             ["abc_classification_product_level_ids"]
        )
```